src/App.jsx

```diff
@@ -0,0 +1,30 @@
+import './App.css'
+
+function App() {
```

Comment:
Since you are using the arrow function, you should also change this component to be arrow function so that you have codebase with same pattern.